### PR TITLE
app/vmalert/datasource/graphite: allow overriding "from" parameter for datasource queries

### PR DESCRIPTION
**Pull request by zekker6**
Fixes construction of URL parameters for graphite render to allow overriding "from" parameter.

See: https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4685

### CODE REVIEW

File: app/vmalert/datasource/vm_graphite_api.go [hunk 1]
```diff
@@ -64,12 +64,14 @@ func (s *VMStorage) setGraphiteReqParams(r *http.Request, query string, timestam
 	}
 	q.Set("format", "json")
 	q.Set("target", query)
-	from := "-5min"
-	if s.lookBack > 0 {
-		lookBack := timestamp.Add(-s.lookBack)
-		from = strconv.FormatInt(lookBack.Unix(), 10)
+	if !q.Has("from") {
```
Comment (hagen1778) [inline app/vmalert/datasource/vm_graphite_api.go:67]: I'd suggest changing the order of applying expected params and extra params. Please see how it is done in prometheus implementation.

Comment (zekker6) [reply app/vmalert/datasource/vm_graphite_api.go]: Updated flow to make it similar to Prometheus one:
1) Sets all default values
2) Merges values with provided `extraParams`

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ The following `tip` changes can be tested by building VictoriaMetrics components
...
 * FEATUTE: [vmalert](https://docs.victoriametrics.com/vmalert.html): support option for "blackholing" alerting notifications if `-notifier.blackhole` cmd-line flag is set. Enable this flag if you want vmalert to evaluate alerting rules without sending any notifications to external receivers (eg. alertmanager). See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4122) for details. Thanks to @venkatbvc for [the pull request](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4639).
 * FEATURE: [vmalert](https://docs.victoriametrics.com/vmalert.html): add unit test for alerting and recording rules, see more [details](https://docs.victoriametrics.com/vmalert.html#unit-testing-for-rules) here. Thanks to @Haleygo for [the pull request](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4596).
+* FEATURE: [vmalert](https://docs.victoriametrics.com/vmalert.html): allow overriding `from` parameter for alerts with `graphite` datasource type. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4685).
```
Comment (hagen1778) [inline docs/CHANGELOG.md:50]:
```suggestion
* FEATURE: [vmalert](https://docs.victoriametrics.com/vmalert.html): allow overriding default GET params for rules  with `graphite` datasource type, in the same way as it happens for `prometheus` type. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4685). 
```

**Review by hagen1778 — APPROVED** [review]
LGTM

**Pull request merged**